entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -e
+
+# Extract the base64 encoded config data and write this to the KUBECONFIG
+echo "$KUBE_CONFIG_DATA" | base64 --decode > /tmp/config
+export KUBECONFIG=/tmp/config
+
+##Install litmus if it is not already installed
+if [ $INSTALL_LITMUS = true ]
+then
+  sh /litmus/installation.sh
+fi
+
+##Select an experiment 
+if [ $EXPERIMENT_NAME = "pod-delete" ]
```

Comment:
You can do like this also:

```sh
EXPERIMENT_URL="/experiments/$EXPERIMENT_NAME/$EXPERIMENT_NAME.sh"
sh $EXPERIMENT_URL
```

Reply:
I will be changing this approach. But can't do like this as for empty `$EXPERIMENT_NAME` it will give error that directory not found but for such case it should pass the code

Reply:
older